### ISSUE TITLE
June 2020

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon#6164]
## Timeline

**June 8 - 12**

- Planning for June Iteration
  - [x] June 8th
- Ship patch release
- Start patch release
- Weekly issue and proposal triage
  - [x] June 12th

**June 15 - 19**

- Ship patch release
- Start patch release
- Weekly issue and proposal triage
  - [x] June 19th

**June 22 - 26**

- Ship patch release
- Start patch release
- Weekly issue and proposal triage
  - [x] June 26th

**June 29 - July 2**

- Ship patch release
- Start endgame (Release candidate, minor release)

**July 6- July 10**
- Ship minor release
- Demo day
  - [ ]  July 8(Wednesday!)
- Retro
  - [ ] July 2
- Planning for 

## October release announcement

Share a release announcement whose content is done in coordination with the broader team. This effort will include team activities to understand what we want to communicate as the top-level changes landing in October.

Owners:
- @joshblack 

Deliverables:
- A blog post that is shared through our Medium publication and is shared through:
  - Slack
  - Twitter

Measurable:
- The degree to which folks are aware that there is a major release happening in October
- The degree to which folks understand the scope of the changes happening in October

## Open Carbon
- Why is there currently a perception that Carbon isn't open?
- Documenting or enabling processes for contribution

### Component index 
@aagonzales @jillianhowarth @janchild 
 - [x] [Data collection](https://github.com/carbon-design-system/carbon-website/issues/1257) @jillianhowarth 
 - [x] [Design work](https://github.com/carbon-design-system/carbon-website/issues/1266) @aagonzales 

### Infrastructure setup
 - [ ] [Carbon Focal and PAL PoC setup activities](https://github.com/carbon-design-system/carbon-website/issues/1256) @shinytoyrobots 

### IA and content strategy plan for site @janchild 
 - [x] Analysis of content needs--identify realignment of existing content plus new content 
 - [x] First pass of content re-positioning, IA
 - [ ] Staging version of a realigned nav

### Elements

Document our elements packages and how to leverage them as standalone efforts or through integration with `carbon-components`.

### Components

- [ ] [Design] Production guidelines https://github.com/carbon-design-system/carbon-website/issues/934 @jeanservaas 
- [ ] *Research - validate code tab documentation updates*
- [ ] [Design] [Add mobile touch screen interactions](https://github.com/carbon-design-system/carbon-website/issues/1264) to updated components @laurenmrice 
- Modal @emyarod (took over for Akira) @aagonzales
  - [ ] Code [PR](https://github.com/carbon-design-system/carbon-website/pull/1126) _*in progress waiting on research to validate_
- Notification  @jillianhowarth @dakahn 
  - [x] Code https://github.com/carbon-design-system/carbon-website/pull/1217
- Button @jillianhowarth @jeanservaas 
  - [x] Usage https://github.com/carbon-design-system/carbon-website/pull/971, https://github.com/carbon-design-system/carbon-website/pull/1306
- Data table  @designertyler @joshblack 
  - [x] Usage https://github.com/carbon-design-system/carbon-website/issues/1063
  - [ ] Code https://github.com/carbon-design-system/carbon/issues/6115 _*in progress waiting on research to validate_
- Content switcher @jillianhowarth @aledavila 
   - [x] Usage https://github.com/carbon-design-system/carbon-website/issues/1207
   - [ ] Code https://github.com/carbon-design-system/carbon-website/pull/1223 _*waiting on research to validate_
- Code snippet @laurenmrice 
   - [x] Usage https://github.com/carbon-design-system/carbon-website/issues/1247
   - [ ] Code

_Next in the list:
Data table(code), UI shell header, List, File uploader, Inline loading, Tabs, Loading, Overflow menu, Text input, Select, Pagination_

### Patterns
- [x] Text toolbar (external contribution design pattern) @laurenmrice 
   - Issue https://github.com/carbon-design-system/carbon/issues/2447  
   - PR: https://github.com/carbon-design-system/carbon-website/pull/1303

### Data visualization
- [x] Design specs for data viz color page redesign
- [ ] Finalize a version of the charts sketch file and make available on the Carbon website. Host the file (in Box?) and update resource cards on the Carbon website. @jeanservaas 
- [x] Merge data viz image update PR [https://github.com/carbon-design-system/carbon-website/pull/1084](https://github.com/carbon-design-system/carbon-website/pull/1084) @jeanservaas 
- [x] Merge demo PR - Updates the chart demos to latest component
[https://github.com/carbon-design-system/carbon-website/pull/987/files](https://github.com/carbon-design-system/carbon-website/pull/987/files)
- [ ] Proposed design: Metering  @designertyler 
[https://github.com/carbon-design-system/carbon-charts/issues/601](https://github.com/carbon-design-system/carbon-charts/issues/601)
- Work with the DB2 team to spec a horizontal stacked bar chart that will meet the needs requested in this issue.
- Specs for chart sub-label
- [ ] Design specs for stacked area chart @designertyler 
[https://github.com/carbon-design-system/carbon-charts/pull/549](https://github.com/carbon-design-system/carbon-charts/pull/549)
- [ ] Facilitate Watson Health chart patterns contribution @designertyler 
- [ ] Thumbnails for chart types 
https://github.com/carbon-design-system/carbon-website/issues/1263

## Ongoing work

- [Treeview component](https://github.com/carbon-design-system/carbon/issues/5180) @emyarod 

## Support

- [x] https://github.com/carbon-design-system/carbon/issues/4902 @joshblack 
- [ ] https://github.com/carbon-design-system/carbon/issues/4839 - @aledavila 
- [ ] https://github.com/carbon-design-system/carbon/issues/4398 @emyarod 
- [ ] [https://github.com/carbon-design-system/carbon/issues/4758](https://github.com/carbon-design-system/carbon/issues/4758) @emyarod 
- [x] https://github.com/carbon-design-system/carbon/issues/6195 - @tw15egan 
- [ ] [https://github.com/carbon-design-system/carbon/issues/6102](https://github.com/carbon-design-system/carbon/issues/6102) - @dakahn 
- [x] [https://github.com/carbon-design-system/carbon/issues/5956](https://github.com/carbon-design-system/carbon/issues/5956) @joshblack
- [ ] https://github.com/carbon-design-system/carbon/issues/6187 @joshblack

**Comment by joshblack** [comment]
Deferred as we are blocked:

## Icons / Pictograms
- Pictograms 32x32 migration
- Pictograms stroke/fill conversion
- Bugs with existing assets are addressed
- Design Kits with updated icons and pictograms
- Do a group activity for the icons to update the metadata 

Deferred due to bandwidth

## Support

- Performance challenges in components that can have large data sets
    - Data table
        - [ ] [https://github.com/carbon-design-system/carbon/issues/6147](https://github.com/carbon-design-system/carbon/issues/6147)
        - [ ] [https://github.com/carbon-design-system/carbon/issues/4338](https://github.com/carbon-design-system/carbon/issues/4338)
    - Multi-select
        - [ ] [https://github.com/carbon-design-system/carbon/issues/2598](https://github.com/carbon-design-system/carbon/issues/2598)
